gravitee-apim-console-webui/src/management/settings/portal-ng/portal-ng.component.scss

```diff
@@ -0,0 +1,63 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+@use '../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.portal {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &__form {
+    &__card {
+      margin-bottom: 16px;
+
+      h3 {
```

Comment:
We should avoid overriding basic styles such as with `h3`. The global styles for `h3` should be inherited.

Reply:
Done